example-auth/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
Could we change this out to be the appwrite png?
<img width="256" alt="favicon" src="https://user-images.githubusercontent.com/3102249/195367364-7b8da211-b1cf-4fd7-9895-4bf42e62cb4f.png">

or the SVG would be even better
![built-with-appwrite](https://user-images.githubusercontent.com/3102249/195367495-66cce189-1f9c-4956-882b-456e9a650073.svg)

Reply:
I replaced Vite's svg with Appwrite's logo from [Appwrite's assets](https://appwrite.io/assets)